frontend/src/metabase/dashboard/components/ClickBehaviorSidebar.jsx

```diff
@@ -593,8 +593,8 @@ function ActionOptions({ dashcard, clickBehavior, updateSettings }) {
               {actions.map(action => (
                 <ActionOption
                   key={action.id}
-                  name={action.card?.name || action.name}
-                  description={action.card?.description || action.description}
+                  name={action.name}
+                  description={action.description}
```

Comment:
🥇

Reply:
Haha, yeah, this diff is pretty enjoyable